CONTRIBUTING.md

```diff
@@ -14,7 +14,11 @@ When `App` from `app.js` was loaded in `index.js`, it called `configureStore` fr
 
 The backend code is briefly summarized below:
 
-**[overview of how the backend code functions]**.
+First of all, raw data is loaded to the database by executing `load_data.py` (as what you can see in `README`). Sample raw data is located in `data/sample/schema/` and `data/sample/article/`. The schema is basically some questions (and child questions) related to an article. `load_data.py` will call data parsers in these files: `parse_schema.py`, `data/parse_schema.py` and `data/parse_document.py`. The two parser functions (i.e. `parse_schema()` and `parse_document()`)in `data/` parses specific articles and schemas related to that article. The parser in `parse_schema.py` (note that it's the one in the root folder!) is an object called `TopicsSchemaParser`, and it loads specific topics and schemas related to the article to our data models in `thresher/models.py`. 
```

Comment:
Should we perhaps rename one of the `parse_schema` files to clarify their meaning?  It's quite confusing right now!